src/Microsoft.Azure.WebJobs.Host/Scale/ConcurrencyOptions.cs

```diff
@@ -147,9 +147,9 @@ public string Format()
             {
                 { nameof(DynamicConcurrencyEnabled), DynamicConcurrencyEnabled },
                 { nameof(MaximumFunctionConcurrency), MaximumFunctionConcurrency },
-                { nameof(TotalAvailableMemoryBytes), TotalAvailableMemoryBytes },
                 // TODO: Once Memory monitoring is public add this back
                 // https://github.com/Azure/azure-webjobs-sdk/issues/2733
+                //{ nameof(TotalAvailableMemoryBytes), TotalAvailableMemoryBytes },
```

Comment:
Missed this when hiding the other memory throttle members (e.g. MemoryThreshold)